### PR TITLE
Feat/delete-watchlist

### LINKED ISSUE #13: Delete watchlist
**Is your feature request related to a problem? Please describe.**
Add a delete watchlist command to allow users to select and delete watchlists.

**Additional context**
Ensure deletion persists between bot sessions.

### PR DESCRIPTION
**Pull request by calebWei**
Implemented command /watchlist_delete [watchlist name] which removes an entire watchlist based on the given name input.

 Closes issue #13

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -76,4 +60,47 @@ async def watchlist_see_all(interaction: nextcord.Interaction):
...
+    watchlist_data["watchlists"].append({'name': arg, 'media': []})
+    watchlist_file = open(WATCHLISTFILENAME, 'w')
+    watchlist_file.write(json.dumps(watchlist_data))
+    watchlist_file.close()
+
+    response = f"Created a new watchlist named {arg}. Add movies and shows to your new watchlist!"
+    await interaction.response.send_message(response)
+
+@bot.slash_command(guild_ids=[GUILD_ID], name="watchlist_delete", description="delete an existing watchlist")
+async def watchlist_create(interaction: nextcord.Interaction, arg):
```
Comment (jess-lowe) [inline bot.py]: Does this need to be renamed to watchlist_delete? there's two watchlist_create

Comment (calebWei) [reply bot.py]: Good find, should be fixed now.

File: bot.py [hunk 2]
```diff
@@ -76,4 +60,47 @@ async def watchlist_see_all(interaction: nextcord.Interaction):
...
+    await interaction.response.send_message(response)
+
+@bot.slash_command(guild_ids=[GUILD_ID], name="watchlist_delete", description="delete an existing watchlist")
+async def watchlist_delete(interaction: nextcord.Interaction, arg):
+    #read the json to get all watchlist list
+    watchlist_file = open(WATCHLISTFILENAME, 'r')
+    watchlist_data = json.load(watchlist_file)
+    watchlist_file.close()
+
+    if len(watchlist_data["watchlists"]) == 0: #Respond if there are no watchlists
```
Comment (jess-lowe) [inline bot.py]: I'm not sure these comments are necessary, code is pretty readable without them :)

Comment (calebWei) [reply bot.py]: That's fair, what about the comments below that? Do you reckon I should remove them too?

Comment (jess-lowe) [reply bot.py]: Yeah I think so :)

**Review by jess-lowe — APPROVED** [review]
lgtm

**Pull request merged**